core/templates/pages/about-page/about-page.import.ts

```diff
@@ -25,6 +25,8 @@ angular.module('oppia', [
   'toastr', 'ui.bootstrap'
 ]);
 
+require('Polyfills.ts');
```

Comment:
[Future work] Why don't we have a common import file which will import all the common libs and can be used in all the page imports?

/cc @srijanreddy98

Reply:
I think we can do that... but in GSoC we are moving to router, so we will have only one root module and all of these would be child modules that are lazy loaded.